invenio_app_rdm/config.py

```diff
@@ -635,6 +640,14 @@ def files_rest_permission_factory(obj, action):
         "content-type": "application/x-dc+xml",
         "filename": "{id}.xml",
     },
+    "marcxml": {
```

Comment:
minor: this is duplicated but will be fixed when https://github.com/inveniosoftware/invenio-app-rdm/pull/2050 is merged and this PR rebased.